edweiss-app/model/events.ts

```diff
@@ -0,0 +1,29 @@
+import { FunctionFolder, FunctionOf, NoResult } from './functions';
+
+namespace Events {
+
+    export interface Event {
+        name: string;
+        time: string;
+        date: string; // format YYYY-MM-DD
```

Comment:
Why use strings here?

Reply:
So you point out an interresting point. As you can see on the PR **Side Note** section. This is not my code, it's Gwenaëlle's. This appeared in my PR as I have updated the model and this got inserted in the `edwiss-app/model`.
Hope you understand.

Reply:
Oops sorry I understand